ledger/ledger-on-memory/README.md

```diff
@@ -0,0 +1,48 @@
+# Overview
+
+This document is to help internal engineers work with `ledger-on-memory`.
+
+`ledger-on-memory` is a key/value-based ledger that uses a simple in-memory map as an
+underlying storage. It uses either H2 or PostgreSQL as its index database.
+
+# Ledger On Memory
+
+To build a fat JAR with the server built from HEAD run
+
+    bazel build //ledger/ledger-on-memory:app_deploy.jar
+
+The application can be run from command line with the following command:
+
+    java -Xmx4G -XX:+UseG1GC -jar bazel-bin/ledger/ledger-on-memory/app_deploy.jar --participant participant-id=foo,port=6865
+
+As run from the main project root directory (adjust the location of the JAR according to 
+your working directory).
+
+## Docker
+
+Ledger On Memory can be put into a Docker container by first building it:
+
+    bazel build ledger/ledger-on-memory:app-image
+
+Then by running the Bazel target that imports the image into your local Docker repository, and then
+runs a container from that image.
+
+    bazel run ledger/ledger-on-memory:app-image
+    
+The container will fail to run, but the image has now been loaded into your local Docker
+repository under the name `bazel/ledger/ledger-on-memory`.
+
+It can then be run using Docker:
+
+    docker run \
+        -it \
+        --rm \
+        --name ledger
+        --network docker_default \
+        -p 6861:6865 
+        bazel/ledger/ledger-on-memory:app-image \
```

Comment:
Shouldn’t this be `bazel-bin` instead of `bazel`?

Reply:
Nope, it's not a filesystem path, it's a Docker image tag. The Bazel rule uses your Bazel target path but prefixes "bazel/" to it.

Reply:
Oh of course, thanks for the explanation!